Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -104,14 +108,15 @@ public extension Conv1D where Scalar.RawSignificand: FixedWidthInteger {
         padding: Padding = .valid,
         dilation: Int = 1,
         activation: @escaping Activation = identity,
+        useBias: Bool = true,
```

Comment:
Can a private variable be used in an initializer like this? I mean will we be able to set it to false externally?

Reply:
Here `useBias` is not the same as the internal flag, it's just to know if it has to pass a bias or nil to the main init.